wiki/Tournaments/UKCC/6/en.md

```diff
@@ -7,14 +7,796 @@ tags:
 
 # UK Community Cup 6
 
+![UKCC6 logo](img/logo.png)
+
 The **UK Community Cup 6** (***UKCC6***) was a British 2v2 double-elimination osu! tournament hosted by ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973), ![][flag_GB] [Chrigarr](https://osu.ppy.sh/users/10111130), ![][flag_GB] [\_Takara](https://osu.ppy.sh/users/11599184) and ![][flag_GB] [Doomsday is Bad](https://osu.ppy.sh/users/3481378). It was the sixth instalment of the UK Community Cup series and the series' third 2v2 competition.
 
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2019-01-12/2019-02-03 |
+| Live drawings | 2019-02-18 |
+| Group stage | 2019-02-23/2019-02-24 |
+| Round of 32 | 2019-03-02/2019-03-03 |
+| Round of 16 | 2019-03-09/2019-03-10 |
+| Quarterfinals | 2019-03-16/2019-03-17 |
+| Semifinals | 2019-03-23/2019-03-24 |
+| Finals | 2019-03-30/2019-03-31 |
+| Grand Finals | 2019-04-06/2019-04-07 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 4 months of osu!supporter, unique profile badge, 1st place trophy and £75 |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of osu!supporter, 2nd place medal and £45 |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 months of osu!supporter, 3rd place medal and £25 |
```

Comment:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 4 months of osu!supporter, unique profile badge, 1st place trophy, and £75 |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of osu!supporter, 2nd place medal, and £45 |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 months of osu!supporter, 3rd place medal, and £25 |
```

Reply:
Resolved with https://github.com/ppy/osu-wiki/pull/6341/commits/682662be8ff6091e687d573febf751a9eece6010

Most prize tables don't include the `and` so just formatted to be consistent with other articles. I originally avoided using the oxford comma since it is very rarely used in British English.